__tests__/data/braze_response_output.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "destinationResponse": {
+      "response": {
+        "attributes_processed": 0,
+        "errors": [
+          {
+            "code": "604"
+          }
+        ],
+        "message": "failed"
+      },
+      "status": 200
+    },
+    "message": "[Braze Response Transfom] Request failed for braze with status: 200",
```

Comment:
`[Braze Response Transfom] Request failed for braze with status: 200`
This is confusing IMO. May be showing it as
`[Braze Response Transfom] Request failed for braze with status: 604` would be better I guess

Reply:
This is addressed